.github/workflows/generate-index.yml

```diff
@@ -56,9 +56,12 @@ jobs:
           cd $GITHUB_WORKSPACE
           task poetry:install-deps
 
-      - name: Generate index
+      - name: Generate plugin firmware index
         run: poetry run ./generator.py -a $(which arduino-cli)
 
+      - name: Generate module firmware index
+        run: poetry run ./generator.py -a $(which arduino-cli) --no-new
```

Comment:
wasn't the new flag introduced called `--new`?

Reply:
Using `--new` sets the flag to `true` (which is actually the default, so the behaviour with no flags is the same), while `--no-new` sets it to false. Should I change the names to make it clearer?